operator/v1/types_ingress.go

```diff
@@ -1649,6 +1649,23 @@ type IngressControllerTuningOptions struct {
 	// +optional
 	TunnelTimeout *metav1.Duration `json:"tunnelTimeout,omitempty"`
 
+	// ConnectTimeout defines the maximum time to wait for
```

Comment:
Whoops, we missed this—[the godoc should use the same capitalization as the json field name](https://github.com/openshift/enhancements/blob/master/dev-guide/api-conventions.md#use-json-field-names-in-godoc): "connectTimeout".  
```suggestion
	// connectTimeout defines the maximum time to wait for
```